pom.xml

```diff
@@ -110,6 +111,11 @@
 				</exclusions>
 			</dependency>
 			<dependency>
+				<groupId>org.springframework.integration</groupId>
```

Comment:
I am not sure why do we need this in parent POM. May be this can be removed with just the version tag on the kafka source POM.